### ISSUE TITLE
Proc macro to generate all queries in a directory

### ISSUE THREAD

**Issue by theduke** [graphql-rust/graphql-client#219]
The CLI already supports generating all queries inside a single file into one module.

It would be great to have this functionality via a proc macro as well.

Eg:

```rust
graphql_schema!( schema = "schema.json", queries = "./queries", module = schema ); 
```

**Comment by theduke** [comment]
Oh I just saw #134 , so this is mostly a duplicate although it's not just relevant to the CLI.

**Comment by tomhoule** [comment]
We would need to decide on the module structure when generating code for multiple operations (see for example https://github.com/graphql-rust/graphql-client/issues/134). I am leaning towards the "one module per operation" solution.

**Comment by tomhoule** [comment]
#229  implements a solution to this. This is released as part of 0.7.0.
